pyproject.toml

```diff
@@ -21,6 +21,7 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
+    "numpy",
```

Comment:
Not sure if we should add a minimum version here, given this is indirectly controlled by the pandas version you use (and adding a version here also means we need to keep it updated)

Reply:
Although I see we are explicitly testing with numpy 1.22 in our minimal env, so you can then just add that here and we have to update the test minimum version anyway.